tutorials/preparing-the-vm.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: page
+title: Preparing the Virtual Machine
+---
+
+Complete the following steps to run Kite example code on a Cloudera Quickstart VM.
+
+* Install a VirtualBox or VMWare [Cloudera QuickStart VM][getvm] version 5.2 or later.
+* In that VM, run the following command from a terminal window. This command clones a local copy of Kite code examples you can build and run.
+
+```bash
+git clone https://github.com/kite-sdk/kite-examples.git
+```
+
+* If you haven't already done so, download the [`kite-dataset` CLI JAR][install-cli].
```

Comment:
> I can't find an example where anyone says "download a command" when they mean "download a JAR."

Fair enough. The real problem is that it is needlessly specific. What happens if we change the install so that it is no longer a JAR? Is there any value to calling it a JAR here? Fix it however you think works best.